viofs/svc/virtiofs.cpp

```diff
@@ -2494,7 +2497,8 @@ NTSTATUS VIRTFS::Start()
     VolumeParams.UmFileContextIsUserContext2 = 1;
 //    VolumeParams.DirectoryMarkerAsNextOffset = 1;
     wcscpy_s(VolumeParams.FileSystemName,
-        sizeof(VolumeParams.FileSystemName) / sizeof(WCHAR), FS_SERVICE_NAME);
+        sizeof(VolumeParams.FileSystemName) / sizeof(WCHAR), 
```

Comment:
Trailing whitespace:
```suggestion
        sizeof(VolumeParams.FileSystemName) / sizeof(WCHAR),
```